src/Cynthia.Card/src/Cynthia.Card.Common/CardEffects/Nilfgaard/Derive/CowCarcass.cs

```diff
@@ -15,6 +15,7 @@ public async Task HandleEvent(AfterTurnOver @event)
             {
                 //await Game.Debug($"牛尸的cd减少啦,之前cd为:{Card.Status.Countdown},之后会在基础上减少1");
                 await Card.Effect.SetCountdown(offset: -1);
+                if (Card.Status.CardRow.IsOnPlace()) return; // we need to chek if it's still alive after updating the countdown
```

Comment:
Why we return here if it is still alive? It will never trigger the following affect below if nothing kills it? Or did I misunderstand something?

Reply:
The bug is caused by a priority issue between cards that have an "AfterTurnOver" event. From what I understand, the cow carcass or incinerating trap will register the countown drop and trigger the effect no matter what, if the turn is over and the card is still on the board. But a card like light longship (which should trigger first, it's on the left) can kill it after the turn is over, but the effect will trigger no matter what. This is why we need to check a second time before triggering the effect that the card is still alive.

Reply:
Actually I spoke too fast, it seems indeed it's missing a ! before IsOnPlace, will check it right now.

Reply:
I checked it and it's going to require more work, this second check replicates the Iris bug where it crashes the game. So the problem is indeed coming from that contradiction caused by the fact that a card be alive then dead "during" the event AfterTurnOver. Thank you for your vigilance.

Reply:
On a second though, actually I think it is intended that the effect is not triggered if it is killed at the end of turn. I mean, if it is killed, it can not take effect, right?

Reply:
> ~On a second thought, actually I think it is intended that the effect is not triggered if it is killed at the end of turn. I mean, if it is killed, it can not take effect, right?~
> 
> I see. Can we just check its life > 0 here?
I tried it and it made the game crash. That's odd because my understanding was that the card would be in a 0hp state before going to the graveyard so the check should work.